plugins/@grouparoo/salesforce/src/migrations/000001-plugin-salesforce-syncMode.ts

```diff
@@ -0,0 +1,49 @@
+export default {
+  up: async function (migration) {
```

Comment:
Let's be sure to test this well. It's critical that Salesforce works as expected here. People have them configured just how they want them.